core/flyout_vertical.js

```diff
@@ -51,7 +52,7 @@ Blockly.VerticalFlyout = function(workspaceOptions) {
 
   Blockly.VerticalFlyout.superClass_.constructor.call(this, workspaceOptions);
   /**
-   * Flyout should be laid out horizontally vs vertically.
+   * Flyout should be laid out vertically.
```

Comment:
Above comment is now misleading since `true` means horizontal... maybe change to `Whether the flyout should be laid out horizontally. Vertical by default.`

Or is this just a change to match upstream?

Reply:
Yup, it's a change to match upstream.  I read it as "This subclass is always laid out vertically (so horizontalLayout is false)."